src/PAModel/PAConvert/Yaml/YamlLexer.cs

```diff
@@ -509,6 +515,25 @@ private YamlToken Unsupported(LineParser line, string message)
             return Error(line, message);
         }
 
+        private YamlToken UnsupportedSingleQuote(LineParser line, int lastSingleQuoteIndex, bool isComponent)
+        {
+            string[] lineSplit = line._line.ToString().Split(new string[] { "As" }, StringSplitOptions.None);
```

Comment:
You'll also need to split on " as " here, I think. Otherwise you could catch the characters "as" is they are part of a control name or the name of a control definition. Think about the case `UnpastedControl as screen` - you'd end up with "Unp", "tedControl ", and " screen" as your splits.

Reply:
Good catch. I shall make the changes